src/components/activity/ActivitySessionItem.tsx

```diff
@@ -35,8 +35,10 @@ const StyledContent = styled.div`
 
 const ActivitySessionItem: React.VFC<{
   activitySessionId: string
+  renderSessionType?: string
+  renderLocation?: React.ReactNode
   renderAttend?: React.ReactNode
```

Comment:
`renderProps` should be a function with props not a component
if we want to use a component, please use `location` instead of `renderLocation`